11-react/02-babel-and-webpack/07-quiz/README.md

```diff
@@ -0,0 +1,108 @@
+# Quiz #1
+
+- Tipo: `quiz`
+- Formato: `self-paced`
+- Duración: `15min`
+
+***
+
+## Objetivos
+
+- En esta sección tendrás la oportunidad de evaluarte para que pongas tus
+  conocimientos a prueba. Es una oportunidad para que te calibres y valides que
+  realmente estás aprendiendo. ¡Mucho éxito!
+
+***
+
+## Preguntas
+
+### 1) ¿Qué es un transpilador?
+
+#### Opciones
+
+1. Es una librería de Javascript
+2. Es una herramienta que nos permite escribir pseudo-código
+3. Es un tipo de compilador capaz de traducir el código de un lenguaje a otro
```

Comment:
Trataría de evitar decir que un transpilador es un compilador (técnicamente no lo es), pero es algo parecido. En vez de "compilar" código en bytecode "transpila" código de un lenguaje a otro. Quizás la respuesta correcta se pueda simplificar a algo como: `Es una herramienta que traduce código de un lenguaje a otro`. Qué piensan?

Como referencia: https://scotch.io/tutorials/javascript-transpilers-what-they-are-why-we-need-them